tests/settings/settings_test.py

```diff
@@ -11,48 +11,67 @@
 from tests.conftest import does_not_raise
 
 
-class TestSettings:
-    def test_to_string(self):
-        # Arrange
-        test_settings = Settings()
+def test_to_string():
+    # Arrange
+    test_settings = Settings()
 
-        # Act
-        result = test_settings.to_string()
+    # Act
+    result = test_settings.to_string()
 
-        # Assert
-        result_list = result.split("\n")
-        assert len(result_list) == 3
-        assert result_list[1] == "kernel options  : {}"
+    # Assert
+    result_list = result.split("\n")
+    assert len(result_list) == 3
+    assert result_list[1] == "kernel options  : {}"
 
-    def test_to_dict(self):
-        # Arrange
-        test_settings = Settings()
 
-        # Act
-        result = test_settings.to_dict()
+def test_to_dict():
+    # Arrange
+    test_settings = Settings()
 
-        # Assert
-        assert result == test_settings.__dict__
+    # Act
+    result = test_settings.to_dict()
 
-    @pytest.mark.parametrize(
-        "parameter,expected_exception,expected_result",
-        [({}, does_not_raise(), "127.0.0.1"), (None, does_not_raise(), "127.0.0.1")],
-    )
-    def test_from_dict(self, parameter, expected_exception, expected_result):
-        # Arrange
-        test_settings = Settings()
+    # Assert
+    assert result == test_settings.__dict__
 
-        # Act
-        with expected_exception:
-            test_settings.from_dict(parameter)
 
-        # Assert
-        assert test_settings.server == expected_result
+def test_is_valid():
+    # Arrange
+    test_settings = settings.Settings()
+
+    # Act
+    result = test_settings.is_valid()
+
+    # Arrange
+    assert result is True
 
 
 @pytest.mark.parametrize(
     "parameter,expected_exception,expected_result",
-    [({}, pytest.raises(SchemaError), False)],
+    [
+        ({}, does_not_raise(), "127.0.0.1"),
+        (None, does_not_raise(), "127.0.0.1"),
+        ({"invalid": True}, pytest.raises(ValueError), "127.0.0.1"),
+    ],
+)
+def test_from_dict(parameter, expected_exception, expected_result):
+    # Arrange
+    test_settings = Settings()
+
+    # Act
+    with expected_exception:
+        test_settings.from_dict(parameter)
+
+    # Assert
+    assert test_settings.server == expected_result
+
+
+@pytest.mark.parametrize(
+    "parameter,expected_exception,expected_result",
+    [
+        ({"include": "bla"}, pytest.raises(SchemaError), False),
+        # ({}, pytest.raises(SchemaError), {}), # Does not work because I don't want to hand in 150 dict keys.
```

Comment:
What options other than passing a large dict do we have?

Reply:
Well actually we could try to improve this line now since we don't need a full dictionary anymore thanks to this PR. However if we want to test a full dict it would be indeed that much settings.

Reply:
In this case I think it's better add the comment about it to prevent any possible attempts to move it on top in future.

Reply:
Then I'd use a smaller dictionary for testing or remove the non-working code completely

Reply:
Okay let me find a good example to uncomment this line.